src/app/app-state-handler.ts

```diff
@@ -32,7 +32,7 @@ class AppStateHandler {
       this.id = powerSaveBlocker.start('prevent-display-sleep');
       logger.info('App-state: Prevent display to sleep, id: ' + this.id);
     } else {
-      if (this.id) {
+      if (this.id !== undefined) {
```

Comment:
so `blocker ID` can be 0 ... interesting :D

Reply:
Yeah, it starts from 0 😀